fr.sgued.bodev.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "fr.sgued.bodev",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "bodev",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```

Comment:
this needs an exception https://github.com/flathub-infra/flatpak-builder-lint/blob/da1bd191e450b326276ebde60e9051568eda4525/flatpak_builder_lint/staticfiles/exceptions.json#L3841-L3842 can you open a PR?